docs/source/telegram.chatboostadded.rst

```diff
@@ -1,6 +1,8 @@
 ChatBoostAdded
 ==============
 
+.. versionadded:: NEXT.VERSION
+
```

Comment:
this should go into the docstring of the class instead

Reply:
The docstring of the class `ChatBoostAdded` should have an entry `versionadded:: …`. There is no need to add that to this rst file as well